src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import "./styles.scss";
+import { IDropdownProps } from "../../types/interfaces";
+import { ReactComponent as Arrow } from "../../assets/arrowDown.svg";
```

Comment:
Nothing serious but I would add a newline after imports.
```suggestion
import { ReactComponent as Arrow } from "../../assets/arrowDown.svg";

```